src/test/java/io/kestra/plugin/singer/taps/PipelinewiseSqlServerTest.java

```diff
@@ -0,0 +1,70 @@
+package io.kestra.plugin.singer.taps;
+
+import com.google.common.collect.ImmutableMap;
+import io.kestra.core.runners.RunContext;
+import io.kestra.core.runners.RunContextFactory;
+import io.kestra.core.utils.IdUtils;
+import io.kestra.core.utils.TestsUtils;
+import io.kestra.plugin.singer.models.DiscoverMetadata;
+import io.kestra.plugin.singer.models.StreamsConfiguration;
+import jakarta.inject.Inject;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+class PipelinewiseSqlServerTest extends TapsTest {
+    @Inject
+    private RunContextFactory runContextFactory;
+
+    @SuppressWarnings("unchecked")
+    @Test
+    void run() throws Exception {
+        PipelinewiseSqlServer.PipelinewiseSqlServerBuilder<?, ? extends PipelinewiseSqlServer.PipelinewiseSqlServerBuilder<?, ?>> builder = PipelinewiseSqlServer.builder()
```

Comment:
```suggestion
        var builder = PipelinewiseSqlServer.builder()
```
This is really where using var makes a difference for readability ;)